collection-scripts/gather_run

```diff
@@ -64,3 +64,25 @@ wait_bg
 
 # Create rotated log symlinks after everything else has finished
 rotated_logs_symlinks
+
+# the path to store the compressed result
+export COMPRESSED_PATH=${COMPRESSED_PATH:-"${BASE_COLLECTION_PATH}"}
+# whether delete to delete or keep the uncompressed files.
+# Defaults to keep them
+export DELETE_AFTER_COMPRESSION=${DELETE_AFTER_COMPRESSION:-0}
+
+# create an easy to download tar.xz from the whole content
+archive="${COMPRESSED_PATH}/must-gather.tar.xz"
+
+tar \
+    --exclude='must-gather.tar.xz' \
+    --warning=no-file-changed --ignore-failed  \
+    -cJf \
+    "${archive}" "${BASE_COLLECTION_PATH}" || true
+
+echo "The ${archive} now can be attached to the support case."
+
+if [[ ${DELETE_AFTER_COMPRESSION} -eq 1 ]]; then
```

Comment:
While reading this I was wondering if we can have an additional small iteration to add the description of these environment variables in the related Readme section [1].

[1] https://github.com/openstack-k8s-operators/openstack-must-gather/?tab=readme-ov-file#customize-gathered-data

Reply:
good point. Done.